examples/python/phase_estimation.py

```diff
@@ -44,7 +44,7 @@
 #
 # $$
 # \begin{equation}
-# QFT : |j\rangle\ \longmapsto \frac{1}{\sqrt{N}} \sum_{k=0}^{N - 1} e^{2 \pi ijk/N}|k\rangle, \quad N= 2^m
+# QFT : |j\rangle\ \longmapsto \frac{1}{\sqrt{N}} \sum_{k=0}^{N - 1} e^{2 \pi ijk/N}|k\rangle, \quad N= 2^n
```

Comment:
The `n` here is presumably the same `n` as talked about a couple of paragraphs below, right? ("We can build the circuit for the `n` qubit QFT using `n` Hadamard gates...")

Reply:
Yes, that's correct

Reply:
(It took me a little while comparing this to https://en.wikipedia.org/wiki/Quantum_Fourier_transform#Definition, and the `ijk` part confused me a little because usually those three indices would be indicating the same sorts of thing - except here `i` is the square root of -1, `j` is a some part of a quantum state and `k` is an index for the sum.)

Reply:
But if that's notation that we've used elsewhere, such as in other papers, then that's good to go with - and the important change (adding the `1 / sqrt(N)` constant) is in.